src/runtime/getPositions.ts

```diff
@@ -370,6 +370,11 @@ export async function getPositions(
             // TODO: We'll probably need to allow hooks to specify the app id themselves
             const { sourceAppId } = tokenDefinition
             const hook = hooksByAppId[sourceAppId]
+            if (!hook.getAppTokenDefinition) {
+              throw new Error(
```

Comment:
Hmm, thinking twice about this...
Throwing here will avoid resolving other tokens definitions, right?
So we could be missing some positions?

Reply:
Yes it will fail the whole resolution.

It may not be ideal, but I think it's ok for now, as I prefer that this doesn't go unnoticed.